### ISSUE TITLE
STARTTLS

### ISSUE THREAD

**Issue by raddzz** [panique/php-login-advanced#74]
Hey, I'm unable to get this working using STARTTLS instead of SSL, any ideas?

**Comment by areebmajeed** [comment]
Does this work?


define("EMAIL_USE_SMTP", true);
define("EMAIL_SMTP_HOST", "smtp.sparkpostmail.com");
define("EMAIL_SMTP_AUTH", true);
define("EMAIL_SMTP_USERNAME", "SMTP_Injection");
define("EMAIL_SMTP_PASSWORD", "xxxxxxxxxx");
define("EMAIL_SMTP_PORT", 587);
define("EMAIL_SMTP_ENCRYPTION", "starttls");

**Comment by raddzz** [comment]
Nope @areebmajeed

**Comment by areebmajeed** [comment]
It should work as this is the same I use. Please try enabling debugging mode and paste what it echoes.

**Comment by panique** [comment]
Hey, your problem is probably outside of this script (which is btw more than 3 years old), can you check if your SMTP provider really offers STARTTLS on also on exactly that port ?

**Comment by areebmajeed** [comment]
That's right! As Chris said, you need to confirm the port and the SMTP credentials, you can do this just by enabling the debugging in PHPMailer.
